Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -3662,6 +3666,20 @@ - (void)_frameInfoFromHandle:(_WKFrameHandle *)handle completionHandler:(void (^
     });
 }
 
+- (BOOL)_showsColorInToolbar
+{
+    return _showsColorInToolbar;
+}
+
+- (void)_setShowsColorInToolbar:(BOOL)showsColorInToolbar
+{
+    _showsColorInToolbar = showsColorInToolbar;
+
+    [self _setUsesAutomaticContentInsetBackgroundFill:showsColorInToolbar];
```

Comment:
Hm... it doesn't seem great that the side effect of this SPI is that it changes the value of another SPI underneath the client. Instead, there should be a new helper method like `- (BOOL)_prefersHardScrollPocketStyle` that would return `self. _usesAutomaticContentInsetBackgroundFill || self._shouldSuppressTopFixedColorExtensionView`?